microsetta_public_api/api/tests/test_integration.py

```diff
@@ -201,6 +221,51 @@ def test_group_summary(self):
 class AllIntegrationTest(
         AlphaIntegrationTests,
         TaxonomyIntegrationTests,
+        MetadataIntegrationTests,
         ):
 
-    pass
+    def test_metadata_filter_on_taxonomy(self):
+        response = self.client.get('/api/metadata/sample-ids?taxonomy=table2')
+        print(response.data)
+        self.assertEqual(response.status_code, 200)
+        obs = json.loads(response.data)
+        self.assertCountEqual(['sample-1', 'sample-2', 'sample-3'],
+                              obs['sample_ids'])
+
+    def test_metadata_filter_on_taxonomy_and_age_cat(self):
+        response = self.client.get(
+            '/api/metadata/sample-ids?taxonomy=table2&age_cat=50s')
+        print(response.data)
```

Comment:
...seems to be a few of these

Reply:
Can you tell which tests I broke during development 🙃 
Fixed in https://github.com/biocore/microsetta-public-api/pull/19/commits/b91aa55c753668901aa122e3bb0c4bb7907d0190